src/reactviews/pages/QueryResult/table/plugins/ColumnMenuPopup.tsx

```diff
@@ -530,7 +531,17 @@ export const ColumnMenuPopup: React.FC<ColumnMenuPopupProps> = ({
             onMouseDown={(e) => e.stopPropagation()}
             onKeyDown={handleRootKeyDown}>
             <div className={styles.titleBar}>
-                <Text className={styles.sectionHeading}>Sort</Text>
+                <Text className={styles.sectionHeading}>
+                    {locConstants.queryResult.sort}
+                    <span
+                        style={{
+                            fontSize: "8px",
+                            fontWeight: "100",
+                            marginLeft: "6px",
+                        }}>
+                        (Shift + Alt + O)
```

Comment:
Even though thye're not localized, I think we should keep all keyboard shortcut strings in a central spot (shortcutConstants.ts?).  Since we have different ways to hint at the shortcut, centralizing the strings would be especially nice for traceability, IMO.